clientv3/README.md

```diff
@@ -0,0 +1,64 @@
+# etcd/clientv3 [![Godoc](http://img.shields.io/badge/go-documentation-blue.svg?style=flat-square)](https://godoc.org/github.com/coreos/etcd/clientv3)
+
+`etcd/clientv3` is the official Go etcd client for v3.
+
+
+
+## Install
+
+```bash
+go get github.com/coreos/etcd/clientv3
+```
+
+
+## Accessing etcd
+
+Create client using `clientv3.New`:
+
+```go
+cli, err := clientv3.New(clientv3.Config{
+	Endpoints:   []string{"localhost:12378", "localhost:22378", "localhost:32378"},
+	DialTimeout: 5 * time.Second,
+})
+if err != nil {
+	log.Fatal(err)
+}
+defer cli.Close()
+```
+
+etcd v3 uses [`gRPC`](http://www.grpc.io) for remote procedure calls. And `clientv3` uses
```

Comment:
i do not think we need to leak the internal implementation details, at least for now. 

Jut say then need to close client after using it from etcd client perspective?